.github/workflows/python-publish.yaml

```diff
@@ -0,0 +1,28 @@
+name: python-publish
+on:
+  workflow_dispatch:
+    inputs:
+      library-version:
+        required: true
```

Comment:
as optional parameters, you can add destination repository, username, and password.
It can help to test too, what do you think?

Reply:
I added the `pypi-repository-url` and `pypi-user` as optional parameters.
It seems that secrets cannot be referenced from the `inputs` section.